legacy/components/Send/SendScreen.js

```diff
@@ -259,25 +259,23 @@ class SendScreenLegacy extends Component<LegacyProps, State> {
 
       const fee = (await transactionData.fee()).getDefault()
 
-      const defaultAssetAmount = selectedTokenMeta.isDefault
+      let isDefaultAssetSelected = selectedTokenMeta.isDefault;
```

Comment:
why is this `let`?

Reply:
ah, automatically generated, haven't noticed. Good catch!